dinner_party.rb

```diff
@@ -0,0 +1,68 @@
+Dinner_party = {
```

Comment:
This is strange that this wouldn't work lowercase, that should be how it is... If you experience this again, let's take a look at it.